rxconfig/application.yaml.template

```diff
@@ -29,3 +29,6 @@ public-key: >
   ielygT6Yl3tNqGGnxm7moXO8+y5uJZymoMqEhV5OnlolpAb/VuGZviv932fWzEMR
   jwIDAQAB
   -----END PUBLIC KEY-----
+
+auth:
+	token: eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJFdGVuZG9SWCBBdXRoIiwiaWF0IjoxNjk1MTUzMDI0LCJhZF91c2VyX2lkIjoiMTAwIiwiYWRfY2xpZW50X2lkIjoiMCIsImFkX29yZ19pZCI6IjAiLCJhZF9yb2xlX2lkIjoiMCIsInNlYXJjaF9rZXkiOiJzZWFyY2hLZXkiLCJzZXJ2aWNlX2lkIjoic2VydmljZUlkIn0.Fc1ME312-KpQNV2a01f12chO7nZ93nDP2aKkeEorX5i7fYP8KASuDrqrI_AXG-j-U_WpO-Dh1fGOJEnbhWqYhTISCD9FQ3FW2wc81jnZDbnxmk75CtgyWu3KUWHxxuwuUExgWz7uEAgdiIytPQJkvpG_uGP5G51RpFMvaQSI6WAEXviJ78r-EP1cigiOpLsBHShx_BXZRIkAXuAFqZMrKUz51LVMSb7XZkeO5gyiIv2QQf5QZ_5-QydDtwe-LVr7dA7JFCIgZT2ax5bKRpFy8I08tqHaJNDpr6ea0-cQqVJ4isssDl5f6CtvU366a_hYBJhm3syuHWqgqFCKqKTs_w
```

Comment:
In which cases is this token necessary? It is preferable to have a token for each service in order to granularise security.

Reply:
I added it because on com/etendorx/auth/auth/TokenController.java a Vale annotation is used to take the token from the yaml

```java
  @Value("${auth.token:}")
  String token;
```

Is that okay or should I change it to take it from another place?